plugins/base/src/main/kotlin/translators/isException.kt

```diff
@@ -1,6 +1,15 @@
 package org.jetbrains.dokka.base.translators
 
 import org.jetbrains.dokka.links.DRI
+import org.jetbrains.dokka.model.AncestryNode
+import org.jetbrains.dokka.model.TypeConstructor
+
+internal fun AncestryNode.typeConstructorsBeingExceptionsOrNull(): List<TypeConstructor>? {
```

Comment:
I think it's widely accepted that returning an empty list instead of a null is a good practice

Is there maybe a good reason to not do that here? If not, what do you think about returning an empty list instead?

Reply:
It is laterly chained in functions that return Extra or null.
```
        private fun AncestryNode.exceptionInSupertypesOrNull(): ExceptionInSupertypes? =
            typeConstructorsBeingExceptionsOrNull()?.let { ExceptionInSupertypes(it.toSourceSetDependent()) }
```
I can make it emptyList and test for that using `takeIf { it.isNotEmpty() }` if it is worth it,

```
        private fun AncestryNode.exceptionInSupertypesOrNull(): ExceptionInSupertypes? =
            typeConstructorsBeingExceptions().takeIf { it.isNotEmpty() }?.let { ExceptionInSupertypes(it.toSourceSetDependent()) }
```

Reply:
Ok, will change